src/php/Analyzer/AnalyzeTuple.php

```diff
@@ -36,58 +37,71 @@ final class AnalyzeTuple
 {
     use WithAnalyzer;
 
+    private const EMPTY_SYMBOL_NAME = '';
+
     /** @throws AnalyzerException|PhelCodeException */
-    public function __invoke(Tuple $tuple, NodeEnvironment $env): Node
+    public function toNode(Tuple $tuple, NodeEnvironment $env): Node
     {
-        if (!$tuple[0] instanceof Symbol) {
-            return (new InvokeSymbol($this->analyzer))($tuple, $env);
-        }
+        $symbolName = $this->getSymbolName($tuple);
+        $symbol = $this->createSymbolByName($symbolName);
+
+        return $symbol->toNode($tuple, $env);
+    }
 
-        switch ($tuple[0]->getFullName()) {
+    private function getSymbolName(Tuple $tuple): string
+    {
+        return isset($tuple[0]) && $tuple[0] instanceof Symbol
+            ? $tuple[0]->getFullName()
+            : self::EMPTY_SYMBOL_NAME;
+    }
+
+    private function createSymbolByName(string $symbolName): TupleSymbolToNode
```

Comment:
I would call it `createSymbolAnalyzerByName`, since it doesn't create a Symbol but an Analyzer.